spec/importers/com_data/com_effort_xml_builder_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'importers/importers_helper'
+
+RSpec.describe ComData::ComEffortXmlBuilder do
+
+  let(:data_sets) do
+    [{'FACULTY_USERNAME' => 'lskywalker',
+      'FACULTY_NAME' => 'Skywalker Luke',
+      'COURSE' => 'the Force',
+      'COURSE_YEAR' => '1976-1977',
+      'EVENT_TYPE' => 'Lecture',
+      'EVENT' => 'FTF REQ Various Rooms 10-12 PBL - EndoRepro PBL 1402 - Thyroid',
+      'EVENT_DATE' => '5/25/77 10:00',
+      'UME_CALCULATED_TEACHING_WHILE_NON_BILLING_EFFORT__HRS_' => 2},
+      {'FACULTY_USERNAME' => 'hgranger',
+      'FACULTY_NAME' => 'Granger Hermione',
+      'COURSE' => 'Potions',
+      'COURSE_YEAR' => '1997-1998',
+      'EVENT_TYPE' => 'Sm Grp Facilitation',
+      'EVENT' => 'FTF REQ Various Rooms 10-12 PBL - EndoRepro PBL 1402 - Thyroid',
+      'EVENT_DATE' => '6/26/97 9:45',
+      'UME_CALCULATED_TEACHING_WHILE_NON_BILLING_EFFORT__HRS_' => 7}
+    ]
+  end
+
+  let(:xml_builder_obj) {ComData::ComEffortXmlBuilder.new}
+
+  describe '#build_xml_effort' do
+    it 'should return a list with an xml of INSTRUCT_TAUGHT records' do
+      data_sets.each do |row|
+        faculty = FactoryBot.create :faculty, com_id: row['FACULTY_USERNAME'], college: 'MD'
```

Comment:
I think we should add to this test a case where a faculty does not have 'MD' as the college, and hence gets excluded from the xml.  Not sure how often (if ever) this might happen, but it is part of our xml builder logic, so I think it should be tested.

Reply:
[56f1487](https://github.com/psu-libraries/fams_tools/pull/46/commits/56f1487aa801edf190e84c0f7d08a25123573de0)